alf/trainers/policy_trainer.py

```diff
@@ -266,11 +267,8 @@ def _step(algorithm, env, time_step, policy_state, epsilon_greedy, metrics):
     policy_step = algorithm.predict_step(transformed_time_step, policy_state,
                                          epsilon_greedy)
     next_time_step = env.step(policy_step.output)
-
-    exp = alf.data_structures.make_experience(time_step, policy_step,
-                                              policy_state)
     for metric in metrics:
-        metric(exp)
+        metric(time_step)
```

Comment:
The ```metric()``` assumes ```Experience``` as input. Might be better to follow this assumption?

Reply:
It's already changed in one PR. Only exp replay buffer's observe will take exps.

Reply:
I see. Then it's good.